README.md

```diff
@@ -42,12 +42,15 @@ jobs:
           GIST_ID: 123abc
 ```
 
-2. 💰 Profit
+2. Create a [secret](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets) 🔑 by going to **GitHub repo > Settings > Secrets > New secret** with the following:
+- Name: `GH_PAT`.
+- Value: The token with the `gist` scope generated previously.
+
+3. 💰 Profit
 
 ### Environment variables & secrets
 
 - **GIST_ID:** The ID portion from your gist url `https://gist.github.com/matchai/`**`6d5f84419863089a167387da62dd7081`**.
-- **GH_PAT:** The GitHub token generated above.
```

Comment:
Can we keep this in? It's still a helpful reference for the requirements of the `.yml` file.

Reply:
Alright, sounds good to me!